index.html

```diff
@@ -1229,6 +1229,34 @@ <h2>A project for first-time contributions</h2>
           </div>
         <!-- ________ Janumala Akhilendra card END ________  -->
 
+	      
+	 <!-- ________ Amrutha Sri  card START ________  -->
+         <div class="card">
+            <p class="name">Amrutha Sri</p>
+            <p class="contact">
+              <i class="fab fa-twitter"></i>
+              <a href="#" target="_blank"></a>
```

Comment:
Replace `USERNAME` with your twitter username. If you don't have twitter, replace `twitter` with `github` and put your github username instead of twitter username.
```suggestion
              <a href="https://twitter.com/USERNAME" target="_blank">@USERNAME</a>
```